starknet_py/tests/e2e/fixtures/devnet.py

```diff
@@ -47,6 +47,8 @@ def start_devnet():
         str(1),
         "--seed",  # generates same accounts each time
         str(1),
+        "--timeout",
+        "1000",
```

Comment:
Why is such a long timeout needed?

Reply:
I suppose it's from local testing. I think we could get rid of it. Is this change necessary?